doc: Update docs with Cellular monitor and Serial Terminal apps (#12395)
nrfconnect/sdk-nrf: doc/nrf/test_and_optimize/testing.rst

#. Click :guilabel:`OK` to view the logging output from the device.

.. _serial_terminal_connect:

How to connect with Serial Terminal
***********************************

> To connect to nRF91 Series devices (for example, the nRF9161 DK, the nRF9160 DK or Thingy:91), you can also use the `nRF Connect Serial Terminal`_ app, which is implemented in `nRF Connect for Desktop`_.

Reviewer: ```suggestion
To connect to nRF91 Series devices (for example, the nRF9161 DK, the nRF9160 DK or Thingy:91), you can use the `nRF Connect Serial Terminal`_ app, which is part of in `nRF Connect for Desktop`_.
```
Doesn't the nRF Connect Serial Terminal app work for all nRF series? Why is only nRF91 mentioned here specifically?
